shell/common/engine.cc

```diff
@@ -69,6 +70,9 @@ Engine::Engine(Delegate& delegate,
       settings_.isolate_create_callback,     // isolate create callback
       settings_.isolate_shutdown_callback    // isolate shutdown callback
   );
+
+  pointer_data_dispatcher_ =
+      platform_view.MakePointerDataDispatcher(*animator_, *runtime_controller_);
```

Comment:
Done. I sent in a `std::function<std::unique_ptr<PointerDataDispatcher>(...)>` instead of a `PointerDataDispatcher` because `Animator&` and `RuntimeController&` must be initialized at construction.